Dockerfile

```diff
@@ -57,15 +63,27 @@ ARG PACKAGER_NAME
 COPY --from=goversioninfo /out/goversioninfo /usr/bin/goversioninfo
 # in bullseye arm64 target does not link with lld so configure it to use ld instead
 RUN [ ! -f /etc/alpine-release ] && xx-info is-cross && [ "$(xx-info arch)" = "arm64" ] && XX_CC_PREFER_LINKER=ld xx-clang --setup-target-triple || true
-RUN --mount=type=bind,target=.,ro \
+RUN --mount=type=bind,target=. \
```

Comment:
What parts need to be writable here? (could that be solved with a separate `tmpfs` mount?

Reply:
Oh it's just `ro` is the default behavior so don't think we need to keep it